### DISCUSSION TITLE
aerc

### DISCUSSION THREAD

**Discussion by mpldr** [catppuccin/catppuccin#1989] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### aerc

aerc is a TUI email client, similar to mutt, but built around git. see also: [website](https://aerc-mail.org)

### How can the suggested port be themed?

aerc provides the ability to have stylesets. It is possible to just add a catpuccin styleset ([manpage](https://man.archlinux.org/man/community/aerc/aerc-stylesets.7.en))

### Any additional comments?

As a maintainer I would also be willing to add it as one of the default-stylesets.

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by samnystrom** [comment]
This has been on my todo list for a while, you beat me to it! Let me know if you want any help with the port.

**Reply by mpldr** [reply]
Gladly :) My current mocha styleset is the following (still very much WIP)

```ini
default.fg=#cdd6f4

tab.bg=#74c7ec
tab.fg=#1e1e2e
tab.selected.bg=#89b4fa
tab.selected.bold=true
title.bg=#74c7ec
border.bg=#74c7ec

msglist_default.fg=#cff6d4
msglist_unread.bold=true
msglist_unread.fg=#89dceb
msglist_flagged.fg=#f9e2af
msglist_flagged.bg=#f38ba8
msglist_flagged.bold=true
msglist_result.fg=#94e2d5
msglist_result.bold=true
msglist_deleted.dim=true
msglist_*.selected.underline=true

dirlist_*.selected.reverse=true

selector_focused.underline=true

statusline_default.bg=#74c7ec
statusline_default.fg=#cff6d4
statusline_error.bg=#f38ba8
statusline_error.fg=#1e1e2e
statusline_error.bold=true
statusline_success.bg=#94e2d5
statusline_success.fg=#1e1e2e

completion_pill.bg=#9399b2
completion_pill.fg=#74c7ec
completion_pill.selected.bg=#74c7ec
completion_pill.selected.fg=#9399b2
```

**Reply by samnystrom** [reply]
That looks good! You can use [puccinier](https://github.com/catppuccin/toolbox#%EF%B8%8F-puccinier) to generate the other flavours.

**Comment by samnystrom** [comment]
@mpldr are you still planning to work on this? If not, I'd be happy to make the port.

**Reply by mpldr** [reply]
The changes mentioned [here](https://github.com/catppuccin/catppuccin/discussions/1989#discussioncomment-5644319) are not the best thinking about it again, so you are probably better off just using the originally posted.

**Comment by mpldr** [comment]
It's currently on the very low boiler where I update it whenever I notice 
something missing, so feel free to go ahead (a link would be appreciated)

-- 
Moritz Poldrack
https://moritz.sh

**Reply by samnystrom** [reply]
Okay, I should be able to get started soon. Have you updated the styleset since you posted it here?

**Comment by mpldr** [comment]
I think I have made some changes, yes. I can update you on that in a bit.

-- 
Moritz Poldrack
https://moritz.sh

**Comment by samnystrom** [comment]
I've created the port at https://github.com/Stonks3141/ctp-aerc. I ended up changing a few things and modeling the theme after the Helix port. There are still a few bits that are unthemed because I don't know what the option changes, like the completion options.

**Reply by mpldr** [reply]
Thanks for your port :)

**Comment by samnystrom** [comment]
https://github.com/catppuccin/catppuccin/issues/2113

**Comment by sgoudham** [comment]
Closing as it's now merged into the organisation at https://github.com/catppuccin/aerc
